_posts/2018-09-28-haskell-in-production-a-ghc-upgrade-success-story.markdown

```diff
@@ -1,3 +1,4 @@
+by Trevis Elser (@telser)
```

Comment:
I think this should go below the title.

Reply:
What do you mean? It already is :stuck_out_tongue_closed_eyes: